src/main/java/org/sagebionetworks/bridge/sdk/UserSession.java

```diff
@@ -16,26 +26,25 @@
     private final String sessionToken;
     private final SharingScope sharingScope;
     private final boolean authenticated;
-    private final boolean consented;
-    private final boolean signedMostRecentConsent;
     @JsonDeserialize(using=DataGroupsDeserializer.class)
     @JsonSerialize(using=DataGroupsSerializer.class)
-    private DataGroups dataGroups;
+    private final DataGroups dataGroups;
+    @JsonDeserialize(keyUsing = SubpopulationGuidKeyDeserializer.class)
+    private final Map<SubpopulationGuid,ConsentStatus> consentStatuses;
 
     @JsonCreator
     private UserSession(@JsonProperty("username") String username, @JsonProperty("sessionToken") String sessionToken,
-            @JsonProperty("authenticated") boolean authenticated, @JsonProperty("consented") boolean consented,
-            @JsonProperty("sharingScope") SharingScope sharingScope,
-            @JsonProperty("signedMostRecentConsent") boolean signedMostRecentConsent,
-            @JsonProperty("dataGroups") DataGroups dataGroups) {
+            @JsonProperty("authenticated") boolean authenticated, @JsonProperty("sharingScope") SharingScope sharingScope,
```

Comment:
Nit: could you put this on two lines for easier reading. I thought you'd missed sharingScope until I looked closer.